### PR TITLE
DP-1.3 TestPlan: QoS ECN feature config - parameter adjustment

### PR DESCRIPTION
**Pull request by rszarecki**
1. Change in existing test to reflect current needs - max-threshold = min-threshold = 80KB
2. Proposed additiona tests plan marked [TODO] for alternative configuration. No code for this.

### CODE REVIEW

**Comment by pranav-jnpr** [comment]
Junos does not natively support specifying queue fill level in absolute values, it supports implementation of relative percentage values (as added to the OC model using [PR #688] (https://github.com/openconfig/public/pull/688)). We recommend setting MINTH & MAXTH values in percentage using the fields defined through PR688.

The temporal DBB value of ~640 microseconds translates to around ~2.56% of the DBB (considering 25ms of DBB for the PTX10008 platform). Hence, to meet this requirement the specification of min-threshold of 2% and max-threshold of 3% is recommended.

File: feature/qos/tests/qos_ecn_config_test/README.md [hunk 1]
```diff
@@ -8,45 +8,59 @@ Verify QoS ECN feature configuration.
...
-        *   qos/queue-management-profiles/queue-management-profile/wred/uniform/config/max-drop-probability-percent
-
-*   Interfaces
-
-    *   Validate ECN profile can be applied under output interface queue using
-        OC config path:
-        *   /qos/interfaces/interface/output/queues/queue/config/queue-management-profile
+        Test case|min-threshold | max-threshold | enable-ecn | drop  | weight  | max-drop-probability-percent
+        |--------|------------- | ------------- | ---------- | ----- | ------- | ----------------------------
+        |#1      |80KB          |80KB*          | true       | false | not set | 100
```
[the remark this answers is not in the document]
Comment (sachendras) [reply feature/qos/tests/qos_ecn_config_test/README.md:20]: Also, do we know if all the different NOS that we deal w/ has support for vertical buffer utilization cut-off line? It is possible that some implementations need different values for min and max thresholds. What are the possibilities of making this a little more flexible by keeping these values really close to each other as well when not supported?

File: feature/qos/tests/qos_ecn_config_test/README.md [hunk 2]
```diff
@@ -8,45 +8,59 @@ Verify QoS ECN feature configuration.
...
+        |[TODO]#2      |3.125MB       | 6.250MB       | true       | false | not set | 100
+        |[TODO]#3      |1%            | 2%            | true       | false | not set | 100
+
+    * 6.25MB max-treshhold is selected as it represents ~500 mico-seconds of Delay bandwidth Buffer on 100GE interfaces. What is O(2%) of buffer depth, hence allows for micro-burst absorbtion without beackpressing senders and at same time leaves enough DBB for accomodate RTT ECN signaling loop delay in global network for longer burst/congestion.
+
+*   Procedure
+    * Test Case #1 80KB min-threshold equal max-threshold
```
Comment (sachendras) [inline feature/qos/tests/qos_ecn_config_test/README.md:27]: Should one of the line item be
- Ensure that at buffer utilization below the min-threshold, packets are not ECN CE marked
- Ensure that at buffer utilization between min and max thresholds, 100% of the packets are ECN marked
- Ensure that at buffer utilization above max-threshold, 100% of the packets are ECN marked

**Review by sachendras — APPROVED** [review]

File: feature/qos/tests/qos_ecn_config_test/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rszarecki) [reply feature/qos/tests/qos_ecn_config_test/README.md]: "*" fixed.

vertical is vertical - so we want to verify it. If vendor needs 79KB/80KB, then deviation is way to go.

File: feature/qos/tests/qos_ecn_config_test/README.md [hunk 3]
```diff
@@ -8,45 +8,59 @@ Verify QoS ECN feature configuration.
 
 *   Connect DUT port-1 to ATE port-1, DUT port-2 to ATE port-2.
 
-*   ECN config:
+*   ECN configuration paramentes:
+
+    *   This test is verifying DUT ability to accept ECN configuration with vertical buffer utilization cut-off line.\ If buffer is utilized below that cut-off value no packet is ECN CE marked.\ If buffer is utilized at of above that cut-off value all packet are ECN CE marked.
```
[the remark this answers is not in the document]
Comment (rszarecki) [reply feature/qos/tests/qos_ecn_config_test/README.md:13]: new line character.

File: feature/qos/tests/qos_ecn_config_test/README.md (hunk 2 above)
Comment (rszarecki) [reply feature/qos/tests/qos_ecn_config_test/README.md:27]: That woud be separate test that is HW specific. 
This one is just update to existing test that can be run on virtual device / KNE

**Pull request merged**